distributed/deploy/local.py

```diff
@@ -27,11 +30,10 @@ class LocalCluster(object):
     ----------
     n_workers: int
         Number of workers to start
+    processes: bool
+        Whether to use processes (True) or threads (False).  Defaults to True
```

Comment:
By the way, why is the default true? Because of better performance on some workloads?